core/src/main/scala/core/database/QueryRunner.scala

```diff
@@ -4,19 +4,20 @@ import scala.concurrent.{ExecutionContext, Future}
 
 import play.api.db.Database
 
+import core.utils.Id
+
 class QueryRunner[DB <: Database](
-    db: Database,
-    ec: ExecutionContext
+  db: Database,
```

Comment:
Features similar to `withConnection` & `withTransaction` could easily be provided without dependency on Play JDBC (which is not standalone and so would pull a significant amount of code): https://github.com/playframework/playframework/blob/38abd1ca6d17237950c82b1483057c5c39929cb4/framework/src/play-jdbc/src/main/scala/play/api/db/Databases.scala#L147